### PR TITLE
Add stop_previous_versions to the gcloud lib

### PR DESCRIPTION
**Pull request by johnnie502**
As title says, this PR add the script

### CODE REVIEW

File: lib/gcloud.sh [hunk 1]
```diff
@@ -35,3 +35,23 @@ delete_old_versions() {
...
+# Stop previous versions of service on Google App Engine.
+#
+# Arguments:
+#   1 the name of the service on GAE
+######################################################
+stop_previous_versions() {
+    echo -e "\nStopping all previous serving versions of $1"
+    # Fetch all previous serving versions sorted by creation time in desc. order.
+    # Remove headings and start from second line.
+    ids=$(gcloud app versions list --service default \
```
Comment (BKorenek) [inline lib/gcloud.sh:49]:
```suggestion
    local ids=$(gcloud app versions list --service default \
```

Comment (BKorenek) [reply lib/gcloud.sh:49]: Aren't we supposed to wrap these guys in double quotes so there's no chance of lines being messed up?

File: lib/gcloud.sh [hunk 2]
```diff
@@ -35,3 +35,23 @@ delete_old_versions() {
...
 }
+
+######################################################
+# Stop previous versions of service on Google App Engine.
+#
+# Arguments:
+#   1 the name of the service on GAE
+######################################################
+stop_previous_versions() {
+    echo -e "\nStopping all previous serving versions of $1"
```
Comment (BKorenek) [inline lib/gcloud.sh:46]: We should establish a local variable for the argument 
`local service="$1"` so it can be used here and in the end of method.

Comment (BKorenek) [reply lib/gcloud.sh:46]: Is this echo entirely necessary? Aren't we echoing before each version as is?

**Review by BKorenek — CHANGES_REQUESTED** [review]

File: lib/gcloud.sh (hunk 2 above)
Comment (Chekote) [reply lib/gcloud.sh:46]: It's necessary because if there are no versions to stop, this will be the only output.

File: lib/gcloud.sh (hunk 1 above)
Comment (Chekote) [inline lib/gcloud.sh:49]: It seems you forgot to use the service parameter and are hard-coding `default` instead.

**Review by Chekote — CHANGES_REQUESTED** [review]

**Review by Chekote — APPROVED** [review]

File: lib/gcloud.sh (hunk 2 above)
Comment (BKorenek) [reply lib/gcloud.sh:46]: seems implicit ;)

**Review by BKorenek — APPROVED** [review]

File: lib/gcloud.sh (hunk 2 above)
Comment (Chekote) [reply lib/gcloud.sh:46]: It's a question of verbosity. Should you only state what you are attempting to do, or what you succeed in doing.

**Pull request merged**